torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -142,8 +142,7 @@ auto CopySlices::apply(variable_list&& inputs) -> variable_list {
       AT_ASSERT(res[i].defined());
       if (i == 0) {
         grad_slice.copy_(res[i]);
-        // NOLINTNEXTLINE(clang-analyzer-cplusplus.Move)
```

Comment:
Pretty sure this was right before, result isn't used at all after this (this branch is never triggered again). I would revert the change for this file as this makes sense and this piece of code seems fairly perf critical.